### PR TITLE
Resolving name conflicts

### PR DESCRIPTION
**Pull request by rishubhjain**
There are name conflicts between attributes derived from master.yaml file and attributes of object derived from _get_object() in tendrl/commons__init__.py

### CODE REVIEW

**Comment by rishubhjain** [comment]
@r0h4n Similar to these there are few others which I can resolve if this issue is legit

File: tendrl/commons/objects/definition/master.yaml [hunk 1]
```diff
@@ -246,7 +246,7 @@ namespace.tendrl:
         sysfs_busid:
           help: "sysfs bus id"
           type: String
-        alignement:
+        alignment:
           help: "alignement"
```
Comment (r0h4n) [inline tendrl/commons/objects/definition/master.yaml]: correct the "help" too

File: tendrl/commons/objects/definition/master.yaml [hunk 2]
```diff
@@ -288,7 +288,7 @@ namespace.tendrl:
         discard_max_bytes:
           help: "discard max bytes"
           type: String
-        discard_zeroes_data:
+        discard_zeros_data:
           help: "discard zeroes data"
```
Comment (r0h4n) [inline tendrl/commons/objects/definition/master.yaml]: fix "help" too

File: tendrl/commons/objects/definition/master.yaml (hunk 1 above)
Comment (rishubhjain) [inline tendrl/commons/objects/definition/master.yaml]: Done

File: tendrl/commons/objects/definition/master.yaml (hunk 2 above)
Comment (rishubhjain) [inline tendrl/commons/objects/definition/master.yaml]: Done, Also there are several such changes, Should I change the others too?

Comment (r0h4n) [inline tendrl/commons/objects/definition/master.yaml]: Yes, as long as they dont break anything

**Pull request merged**